### ISSUE TITLE
Abstract Members Not Properly Validated in Multiple Mixin Composition

### ISSUE THREAD

**Issue by piotrjoniec** [microsoft/TypeScript#62014]
### 🔎 Search Terms

abstract getters members methods mixin

### 🕗 Version & Regression Information

- This is the behavior in every version I tried, and I reviewed the FAQ

### ⏯ Playground Link

https://www.typescriptlang.org/play/?ts=5.9.0-dev.20250707#code/C4TwDgpgBAwg9gOwM7AE4FcDGw6qgXigQgHcoAKAOmoENUBzJALihoRAG0BdASgID4oAbwC+AbgBQEgGboE2AJaIoAWQUAPBQgCCAHgAqUCOuAQEAEySxEKDNlz9yAIxpIILfXyESorJ7ZpsKEwAG1crNU0EIxMzSygXN2EfXz8AoNMUFQhgAAs4c3IeFlsteklfESlfVBz0VGjIrUkqmTlFZSaEACEDGNMLK3hkNCwcVEdE9yhPZN8afzRA4GCwpAiNLX64qym51IX0lczgbLyCopK0MoqoVpq6htVNhBapUPCobtdoUSkAen+UAAcnAjKhULgoLUwLhTOYCFAAGLaACSABkJB91lBUQBbMAhCB4szAGjAJQ9baDZ5RbrkLracjfNw8LxSVqAqAAUQhUMIAHkANISLmghAAWkOSyC2KsAHJ8YTiaTyZTtPKoOY4BArAg4CsFASiSSEIaELkIKgFPC0jKViS8U4rVATmd8gjpJC8atPvLGbpQJA4NIvj9+JQulAAGRhtzyyjkABMAFYAIwpnhYtZWJUm1UUxDaanxRnMn7siQiIA

### 💻 Code

```ts
type Constructor = new (...args: any[]) => {};

function MixinA<T extends Constructor>(base: T) {
  abstract class Mixin extends base {
    abstract testMethod(): string;
  }

  return Mixin;
}

function MixinB<T extends Constructor>(base: T) {
  abstract class Mixin extends base {
    abstract testMethod(): string;
  }

  return Mixin;
}

class Base {}

// No error reported = FAIL
class ImplementationB extends MixinB(MixinA(Base)) {

}

// Error = OK
// Non-abstract class 'ImplementationA' does not implement inherited abstract member testMethod from class 'MixinA<typeof Base>.Mixin & Base'.(2515)
class ImplementationA extends MixinA(Base) {

}
```

### 🙁 Actual behavior

There was no error reported for **ImplementationB**

TypeScript fails to detect unimplemented abstract members when multiple mixins are composed together, even when those mixins declare the same abstract member.

### 🙂 Expected behavior

An error should be reported since **ImplementationB** did not implement the abstract method **testMethod** from MixinA and MixinB.

TypeScript should report an error when a concrete class extends multiple mixins that declare abstract members, and those abstract members are not implemented in the concrete class.

### Additional information about the issue

TypeScript fails to detect unimplemented abstract members when multiple mixins are composed together, even when those mixins declare the same abstract member.

**Comment by Andarist** [comment]
duplicate of https://github.com/microsoft/TypeScript/issues/56738
